app/scripts/modules/google/src/loadBalancer/details/loadBalancerDetail.controller.js

```diff
@@ -118,6 +118,29 @@ module.exports = angular.module('spinnaker.loadBalancer.gce.details.controller',
             return [representativeLb];
           });
 
+      } else if (($scope.loadBalancer.provider === 'gce' || $scope.loadBalancer.type === 'gce') && $scope.loadBalancer.loadBalancerType === 'INTERNAL') {
+        var detailsPromises = $scope.loadBalancer.listeners.map((listener) => {
```

Comment:
Looks like `detailsPromises` is defined up above. `var` isn't block scoped in JS, so switch to `const`.